### PR TITLE
headers format 

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
On this PR:

 - Add missing headers
 - Fix format for some copyright headers
 - format some imports (`goimports` tool)

**Checkoff for all PRs:**

- [x] I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [ ] I have added changes to the [CHANGELOG](https://github.com/singularityware/singularity/blob/development/CHANGELOG.md) and and documentation updates to the [singularityware](https://www.github.com/singularityware/singularityware.github.io) documentation base.
- [ ] I have tested this PR locally with a `make test`
- [x] This PR is NOT against the project's master branch
- [x] I have added myself as a contributor to the [contributors's file](https://github.com/singularityware/singularity/blob/master/CONTRIBUTORS.md)
- [x] This PR is ready for review and/or merge

Attn: @singularityware-admin

### CODE REVIEW

**Comment by yhcote** [comment]
I thought the standard was to use // for the copyrights at the top of the files ???

**Review by tri-adam — CHANGES_REQUESTED** [review]
I agree with @yhcote, let's use a format that is consistent with the Golang source itself:

```
// Copyright (c) 2018, Sylabs Inc. All rights reserved.
// This software is licensed under a 3-clause BSD license. Please consult the
// LICENSE file distributed with the sources of this project regarding your
// rights to use or distribute this software.
```

**Comment by ArangoGutierrez** [comment]
`Go provides C-style /* */ block comments and C++-style // line comments. Line comments are the norm; block comments appear mostly as package comments, but are useful within an expression or to disable large swaths of code.`
[effective_go](https://golang.org/doc/effective_go.html#commentary)

**Comment by tri-adam** [comment]
As it says, 

> Line comments are the norm

These are not package comments, nor an expression, nor a large swath of code to be disabled. Let's keep consistent with the Golang source, please.

**Comment by ArangoGutierrez** [comment]
67703f1 changes all copyright headers to:
```go
// Copyright (c) 2018, Sylabs Inc. All rights reserved.
// This software is licensed under a 3-clause BSD license. Please consult the
// LICENSE file distributed with the sources of this project regarding your
// rights to use or distribute this software.
```
@tri-adam 
ready for a second pass.

File: src/cmd/singularity/cli/signing.go [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-  This software is licensed under a 3-clause BSD license.  Please
-  consult LICENSE file distributed with the sources of this project regarding
-  your rights to use or distribute this software.
-*/
 package cli
 
 import (
 	"fmt"
 
+	"os"
```
Comment (tri-adam) [inline src/cmd/singularity/cli/signing.go]: Remove space above this line?

**Review by tri-adam — CHANGES_REQUESTED** [review]

File: src/cmd/singularity/cli/signing.go (hunk 1 above)
Comment (ArangoGutierrez) [reply src/cmd/singularity/cli/signing.go]: `goimports` ... fixing

**Review by tri-adam — APPROVED** [review]

**Pull request merged**